libsplinter/src/rest_api/auth/rbac/rest_api/resources/roles.rs

```diff
@@ -42,6 +42,12 @@ pub struct RolePayload {
     pub permissions: Vec<String>,
 }
 
+#[derive(Deserialize)]
```

Comment:
Shouldn't this be in a `role.rs` file?

Reply:
Probably somewhere like that, and I refactor it in some commits built on top of this - it's not really role related, either. This probably should be a generalized across all of the API's